### PR TITLE
Cleanup scm mojo while debugging

### PR DESCRIPTION
**Pull request by lkwg82**

### CODE REVIEW

File: pitest-maven/src/main/java/org/pitest/maven/PitMojo.java [hunk 1]
```diff
@@ -47,7 +47,7 @@
    * @parameter expression="${targetClasses}"
    *
    */
-  protected ArrayList<String>         targetClasses;
+  private List<String>         targetClasses;
```
Comment (hcoles) [inline pitest-maven/src/main/java/org/pitest/maven/PitMojo.java]: Unfortunately maven 2 breaks unless given a concrete list type

Comment (lkwg82) [inline pitest-maven/src/main/java/org/pitest/maven/PitMojo.java]: I know and you already merged upgrade to maven 3, see #199 

So this should be not an issue.

Comment (hcoles) [inline pitest-maven/src/main/java/org/pitest/maven/PitMojo.java]: I'd assumed this just enforced the minimum version for building pitest. Does it set a minimum for someone running the built plugin?

Comment (lkwg82) [inline pitest-maven/src/main/java/org/pitest/maven/PitMojo.java]: I think we did not talk about this specific expectation. I'd assume both.

But maybe we can be clear on #202 to meet our both expactations ;).

Comment (hcoles) [inline pitest-maven/src/main/java/org/pitest/maven/PitMojo.java]: Indeed - was planning on merging that in when I'd had chance to look properly at the very slow site report tests. If you could update the PR to add and ignore to them I'll click the merge button now.

**Comment by lkwg82** [comment]
Which pr? This or #202?

**Comment by hcoles** [comment]
@lkwg82 #202

**Comment by hcoles** [comment]
The less immediate feedback from https://hjc.ci.cloudbees.com/job/maven2_triangle_example/configure confirms that maven 2 is still working fine on master. Change to interface types will break it.

**Pull request closed without merging**